src/components/Cart/CartSummary.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { useCart } from '../../contexts/cart-context';
+
+const CartSummary=()=> {
+    const {cartState:{cart}}=useCart();
+    const totalAmt=cart.reduce((acc,cur)=> Number(cur.price) * Number(cur.qty) + acc,0);
+    const totalItem=cart.reduce((acc,cur)=> Number(cur.qty) + acc,0);
+    const deliveryCharges=100;
+    const finalAmt=totalAmt+deliveryCharges;
+
+    console.log(totalAmt);

```

Comment:
you can remove console.log

Reply:
Done [346210e](https://github.com/Manpreet-hub/ecommerce-react/pull/4/commits/346210efcfbdd93bcf04432da991ab1028dd18a2)